pandas/io/formats/info.py

```diff
@@ -148,23 +201,31 @@ def info(
     exceeds_info_cols = col_count > max_cols
 
     def _verbose_repr():
-        lines.append(f"Data columns (total {col_count} columns):")
 
         id_head = " # "
         column_head = "Column"
         col_space = 2
 
         max_col = max(len(pprint_thing(k)) for k in ids)
         len_column = len(pprint_thing(column_head))
-        space = max(max_col, len_column) + col_space
 
         max_id = len(pprint_thing(col_count))
         len_id = len(pprint_thing(id_head))
         space_num = max(max_id, len_id) + col_space
 
-        header = _put_str(id_head, space_num) + _put_str(column_head, space)
+        header = _put_str(id_head, space_num)
+
+        if isinstance(data, ABCDataFrame):
```

Comment:
ok this is better, but how about if you have a base class and DataFrameInfo and SeriesInfo. I think it will be more clear (might be more code, but that's ok)